examples/ExampleApp.py

```diff
@@ -5,8 +5,6 @@
 import subprocess
 from argparse import Namespace
 
-import qdarkstyle
```

Comment:
:man_facepalming: :man_facepalming: :man_facepalming: :man_facepalming:

Reply:
> Black text always shows up best against the yellow-highlighted background, so I can hard-code that

There is nothing hard-coded in Qt about the highlight color (unless you hardcode it yourself).  Generally speaking, the fewer hard specifications on colors for this sort of thing you can do, the better (aka let Qt sort out the OS integration bits).